packages/typescript/tsconfig.browser.json

```diff
@@ -1,7 +1,6 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "outDir": "./dist.browser",
```

Comment:
`outDir` config should live on the user package config. Otherwise, the built files will end up on the `config-typescript` package.